charts/probate-caveats-fe/values.aat.template.yaml

```diff
@@ -2,4 +2,4 @@ nodejs:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
   environment:
-    LAUNCHDARKLY_ENABLED: false
+    LAUNCHDARKLY_ENABLED: true
```

Comment:
this will be OK to merge?

Reply:
Yes just found out this is the reason why my feature toggle wasn't working on AAT last time